example.json

```diff
@@ -0,0 +1,234 @@
+                        "nodata": false
+                    },
+                    {
+                        "value": 2,
+                        "name": "Herbaceous Vegetation",
+                        "description": null,
+                        "title": null,
+                        "color_hint": null,
+                        "nodata": false
+                    },
+                    {
+                        "value": 3,
+                        "name": "Highway",
+                        "description": null,
+                        "title": null,
+                        "color_hint": null,
+                        "nodata": false
+                    },
+                    {
+                        "value": 4,
+                        "name": "Industrial Buildings",
+                        "description": null,
+                        "title": null,
+                        "color_hint": null,
+                        "nodata": false
+                    },
+                    {
+                        "value": 5,
+                        "name": "Pasture",
+                        "description": null,
+                        "title": null,
+                        "color_hint": null,
+                        "nodata": false
+                    },
+                    {
+                        "value": 6,
+                        "name": "Permanent Crop",
+                        "description": null,
+                        "title": null,
+                        "color_hint": null,
+                        "nodata": false
+                    },
+                    {
+                        "value": 7,
+                        "name": "Residential Buildings",
+                        "description": null,
+                        "title": null,
+                        "color_hint": null,
+                        "nodata": false
+                    },
+                    {
+                        "value": 8,
+                        "name": "River",
+                        "description": null,
+                        "title": null,
+                        "color_hint": null,
+                        "nodata": false
+                    },
+                    {
+                        "value": 9,
+                        "name": "SeaLake",
+                        "description": null,
+                        "title": null,
+                        "color_hint": null,
+                        "nodata": false
+                    }
+                ],
+                "post_processing_function": null
+            }
+        ],
+        "mlm:runtime": [
+            {
+                "asset": {
+                    "href": "https://huggingface.co/torchgeo/resnet18_sentinel2_all_moco/resolve/main/resnet18_sentinel2_all_moco-59bfdff9.pth",
+                    "title": null,
+                    "description": null,
+                    "type": null,
+                    "roles": null
```

Comment:
Agree, I think I would add `executable` to represent exported graphs that combine the model weights with model structure to make an executable graph. Do we need `data`? I wonder if folks would get confused with if they should use this for a model file or not

Reply:
And I think for `type` we could add an enum of common formats. since these can dictate how the model should be loaded by the framework. for starters I would add torch formats the most common I've encountered being

`.pth` model weidhts
`.pt` traced model (Torchscript)
`.pt` scripted model (Torchscript)
`.pt2` Dynamo exported model

And to make this general we should also add other framework formats
- .h5 for keras
- whatever tensorflow uses
- formats for scikit-learn, R ML libraries

I'm certainly missing a lot of representation here. we can document in this section that the community should raise an issue to add a media type to the enum. What do you think? is this the correct location to enumerate model formats?